snowfall/training/graph.py

```diff
@@ -6,21 +6,39 @@
 
 
 class TrainingGraphCompiler:
-    def __init__(self, L: Fsa, vocab: SymbolTable, oov: str = '<UNK>'):
-        self.L = L
+    """
+    Compiles and caches the supervision FSAs created for text phrases.
+    The Fsas are created and stored on the CPU and the user is expected to transfer
+    them to GPU themselves (that might change at some point).
+
+    Args:
+        L_inv:
+            An ``Fsa`` that represents the inverted lexicon (L), i.e. has phones as ``symbols``
+            and words as ``aux_symbols``.
```

Comment:
I think it should be reversed, i.e. has word as "symbols" and phones as "aux_symbols"